### ISSUE TITLE
Proposal: About the ACT Rules CG

### ISSUE THREAD

**Issue by WilcoFiers** [auto-wcag/auto-wcag#466]
## Mission Statement

The ACT Rules Community Group (previously known as [Auto-WCAG](https://w3.org/community/auto-wcag/)), is a community set up to document and harmonise the interpretation of WCAG and other web accessibility standards. The ACT Rules Community Group (ACTR Community) does this by bringing together the interpretations behind various automated accessibility tools and testing methodologies. These interpretations are documented as test rules, as defined in the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/).

Implementors and users of test tools and test methodologies work in the ACTR Community to document, and where possible harmonize interpretations of WCAG and other accessibility standards. By publishing these rules, the ACTR Community hopes to motivate other organisations to adopt common interpretations, and abandon uncommon ones. Where consensus is reached, ACT rules will be submitted to the W3C, as a request from the accessibility rule writing community to adopt its rules as a W3C recognised interpretation of WCAG and other accessibility requirements documents.

The ACT Rules Community is not set up to remove any and all differences from accessibility tools and test procedures. There is a great deal of value in their innovative power. Rather it aims to remove unnecessary differences. How accessibility testing is done is irrelevant, and often considered part of an organisation's intellectual property. But knowing exactly when something meets a requirement, and what it does not, should be as clear as we can make it.

## Why Harmonisation Matters

Accessibility compliance is becoming a legal requirement all around the world. This makes it increasingly important for organisations to understand exactly what they need to do to be compliant. Accessibility requirements such as WCAG 2.1 are often written in technology agnostic language. Interpretation exactly what those requirements mean for a specific technology requires interpretation. This interpretation can be different between different testers, between different organisations and in different regions around the world.

Knowing the interpretation that will be used to assess compliance to legal requirements is essential for organisations. Without this it is impossible for them to know whether or not they can be sued or fined. For those operating in different countries, consistency of that interpretation is also important. Inconsistent interpretations can lead to an overload of requirements, possibly even mutually exclusive requirements. The principle is simple, a website that is considered to conform to WCAG 2.1 Level AA in the US, should also be compliant in the EU.

## Who We Are

The ACT Rules Community Group is a group of accessibility tool vendors, test procedure authors, and accessibility test experts. The ACTR Community has no standing in the W3C, and does not develop W3C recommendations or notes. Rather, it researches and documents accessibility interpretations and makes this information available for the future improvement of documents produced by the Accessibility Guidelines Work Group.

The ACTR Community is an open community, meaning that anyone with an interest in our work is welcome to participate. The ACTR Community has three resources that are important to know about. Github, the community group page, and the ACTR Community website.

- [ACTR Website](https://auto-wcag.github.io/auto-wcag) The ACTR Community website is where all the work is published. If you are interested to learn what rules have been agreed upon by the group, and who has implemented those, this is the place to go.

- [Github](https://github.com/auto-wcag/auto-wcag) All work in the community group happens on Github. To participate you will need a Github account and basic familiarity with Github. If you are new to Github, there are several great articles and video's on the web explaining the basics of Github. Here are a few:
  - http://www.madebyloren.com/posts/github-for-writers
  - https://www.youtube.com/watch?v=9XhbYHcaT9k&t=7s

- [W3C Community Page](https://w3.org/community/auto-wcag/) This is the official page at the W3C. From here you can sign up to the community group. This requires a W3C account. If you work for a W3C member organisation, you will need approval from your organisation to join the group. This is also where the mailinglist archive can be found.

The ACTR Community has calls twice a month, once for a US friendly time, and once for an Asia friendly time. To receive the agenda and calendar, you must be a member of the W3C Community Group. This will automatically sign you up for the mailinglist.

-- 

The Auto-WCAG one can be found here:
https://www.w3.org/community/auto-wcag/about-us/

**Comment by EmmaJP** [comment]
My understanding is that the group are trying to reach a common interpretation of WCAG (and similar guidelines) in regard testing for accessibility, and documenting that as a set of test rules, similar to feature files. Each rule states what should be tested (what the rule applies to and what is expected) and provides examples of code that would pass or fail the test. Contributors bring their own understanding and ideas to the conversation, often based on existing work/tools they are involved with, and together we work toward those common interpretations.

The above doesn't imply the same thing.

**Comment by EmmaJP** [comment]
Suggested edits for the Mission Statement marked with bold:

The ACT Rules Community Group (previously known as Auto-WCAG), is a community set up to document and harmonise the interpretation of WCAG and other web accessibility standards **for testing purposes**. The ACT Rules Community Group (ACTR Community) does this by bringing together the **people** behind various automated accessibility tools and testing methodologies. 

**Implementors and users of test tools and test methodologies work in the ACTR Community to document, and where possible harmonise interpretations of WCAG and other accessibility standards. Their harmonised** interpretations are documented as test rules, as defined **by** the ACT Rules Format.

By publishing these rules, the ACTR Community hopes to motivate other organisations to adopt common interpretations (, and abandon uncommon ones - _implicit so not needed_). Where consensus is reached, ACT rules will be submitted to the W3C, as a request from the accessibility rule writing community to adopt its rules as a W3C recognised interpretation of WCAG and other accessibility requirements documents.

The ACT Rules Community is not set up to remove (any and - _not needed_) all differences from accessibility tools and test procedures. There is a great deal of value in their innovative power. Rather it aims to remove unnecessary differences. How accessibility testing is done is irrelevant, and often considered part of an organisation's intellectual property. But knowing exactly when something meets a requirement, and **when** it does not, should be as clear as we can make it.

**Comment by EmmaJP** [comment]
Suggested edits for Why Harmonisation Matters marked with bold:

Accessibility compliance is becoming a legal requirement all around the world. This makes it increasingly important for organisations to understand exactly what they need to do to be compliant. Accessibility requirements such as WCAG 2.1 are often written in technology agnostic language. **Understanding** exactly what those requirements mean for a specific technology requires interpretation. **To date** interpretation **is often** different between different testers, between different organisations and in different regions around the world.

Knowing the interpretation that will be used to assess compliance to legal requirements is essential for organisations. Without this it is impossible for them to know whether or not they can be sued or fined. For those operating in different countries, consistency of that interpretation is also important. Inconsistent interpretations can lead to an overload of requirements, possibly even mutually exclusive requirements. The principle is simple, **for example,** a website that is considered to conform to WCAG 2.1 Level AA in the US, should also be compliant in the EU.

**Comment by EmmaJP** [comment]
The rest is spot on. Hope the comments above are helpful.

**Comment by skotkjerra** [comment]
I like the updated suggestions. Just to be clear, The AGWG should of course only review rules that test for accessibility requirements that originate in W3C, such as WAI-ARIA or WCAG. Harmonization is a clear goal, but there is also value in documenting different, and possibly diverging, approaches to testing a specific accessibility requirement

**Comment by nitedog** [comment]
For the record, I had sent these rather extensive editorial suggestions and comments to @WilcoFiers for consideration: [Mission Statement.docx](https://github.com/auto-wcag/auto-wcag/files/3037313/Mission.Statement.docx)

**Comment by EmmaJP** [comment]
Some very valid points in there @nitedog ... though a lot of changes to weed through.

**Comment by EmmaJP** [comment]
_Suggested Mission Statement combining various points of view, with US spelling and tightened up a bit (so may have missed something someone thinks an important point):_

The ACT Rules Community Group (previously known as Auto-WCAG), is an open forum working to build a common interpretation around W3C accessibility standards, such as WCAG and WAI-ARIA, for testing purposes. 

The ACT Rules Community Group (ACT-R) achieves this by bringing together the people developing, implementing and using various accessibility testing tools and methodologies to document interpretations as test rules, as defined in the ACT Rules Format and reviewed by the community. The process of documenting, sharing and feedback, from different perspectives within the group, builds towards a common understanding and gradual harmonization. By publishing such harmonized test rules, ACT-R hopes to motivate organizations to adopt commonly agreed interpretations.

ACT-R is not set up to remove differences or impose changes on accessibility testing tools and methodologies. There is value in innovation and diverse approaches. Rather it aims to contribute to more consistent results, regardless of how the testing is done. Knowing when something meets a requirement, and when it does not, should be clear and consistent.

**Comment by WilcoFiers** [comment]
@EmmaJP @nitedog Thank you for the feedback. I've made a few more changes, and am going with the following:

> The ACT Rules Community Group (previously known as Auto-WCAG), is an open forum set up to document and harmonise the interpretation of W3C accessibility standards, such as WCAG and WAI-ARIA, for testing purposes.
> 
> The ACT Rules Community Group (ACT-R) achieves this by bringing together the people developing, implementing and using various accessibility testing tools and methodologies to document interpretations as test rules. Test rules are defined using the ACT Rules Format, and reviewed by the community. The process of researching, documenting, and sharing knowledge from different perspectives within the group, builds towards a common understanding. By publishing such test rules, ACT-R hopes to motivate organisations to share their own insights, and adopt commonly agreed test rules.
> 
> ACT-R is not set up to remove differences or impose changes on accessibility testing tools and methodologies. There is value in innovation and diverse approaches. Rather it aims to contribute to more consistent results, regardless of how the testing is done. Knowing when something meets a requirement, and when it does not, should be clear and consistent.

---

I've tweaked the "harmonisation" in the proposal. Given that this is such a hot topic I want to explain that before I post. I don't think we can call rules "harmonised" as soon as we publish them. Auto-WCAG / ACT-R is a relatively small group, almost exclusively contributed to by EU accessibility (tool) experts. I don't believe we can conclude that, when we agree on something, everyone or even a majority of experts will agree with us. My experience with Auto-WCAG is that that's not how it works.

Publishing rules is not the end point of harmonisation, it's the starting point. We've agreed as a group, now let's see what the rest of the world thinks. Calling any rule harmonised before we've looked at what other tool / methodology authors think seems premature. And while it would be great if all major tool devs / methodology authors would contribute to the group, many of them aren't doing so today, and we shouldn't have the mission of harmonisation be dependent on it.


Please give me a thumbs up or thumbs down on this post. I'll give it another day or so.

**Comment by EmmaJP** [comment]
Shadi pointed out that W3C use American spelling ... "harmonize" (I know!)

"Publishing rules is not the end point of harmonisation, it's the starting point." is a great point that would be good to include somehow. Perhaps as the penultimate sentence of the second paragraph.

> ...towards a common understanding. Publishing rules is not an end point for harmonization, it's a starting point. By publishing such test rules, ACT-R hopes to motivate organisations to share their own insights, and encourage adoption of commonly agreed test rules.

**Comment by nitedog** [comment]
@WilcoFiers I like this rewrite very much! Just the spelling issue. Please update the text, and I will check if we can update it on the CG homepage.

**Comment by WilcoFiers** [comment]
@jkodu can you put this into the new website?

**Comment by jkodu** [comment]
Updated about us in new site PR - https://github.com/auto-wcag/auto-wcag/pull/437/
